.github/workflows/codegen-ci.yml

```diff
@@ -132,31 +110,33 @@ jobs:
           path: artifacts
 
       - name: Publish Unit Test Results
-        uses: EnricoMi/publish-unit-test-result-action@v1.6
+        uses: EnricoMi/publish-unit-test-result-action@v1.12
         with:
-          check_name: Test Results
+          check_name: Codegen Tests
           github_token: ${{ secrets.GITHUB_TOKEN }}
           report_individual_runs: true
+          hide_comments: orphaned commits
+          check_run_annotations_branch: '*'
```

Comment:
tweaking a couple new knobs (upgraded from 1.6 to 1.12) that will give us better signal on test results